src/index.js

```diff
@@ -1,5 +1,4 @@
-export { observe, unobserve } from '@nx-js/observer-util';
```

Comment:
This was added since the last release and it is not yet released to npm. It won't cause a breaking change.